octobot_trading/exchanges/config/exchange_settings.py

```diff
@@ -0,0 +1,96 @@
+import octobot_trading.exchanges.config.exchange_test_status as exchange_test_status
+import octobot_trading.exchanges.parser as parser
+
+
+class ExchangeConfig:
+    """
+    override this class if you implement a new exchange connector
+    see CCXTExchangeConfig as an example
+    """
+
+    # override classes if you need a different parser
+    MARKET_STATUS_PARSER: parser.ExchangeMarketStatusParser = None
+    ORDERS_PARSER: parser.OrdersParser = None
+    TRADES_PARSER: parser.TradesParser = None
+    POSITIONS_PARSER: parser.PositionsParser = None
+    TICKER_PARSER: parser.TickerParser = None
+    FUNDING_RATE_PARSER: parser.FundingRateParser = None
+    
+    FUNDING_IN_TICKER: bool = None
+    MARK_PRICE_IN_TICKER: bool = None
+    FUNDING_WITH_MARK_PRICE: bool = None
+    MARK_PRICE_IN_POSITION: bool = None
+
+    def __init__(self, exchange_connector):
+        self.set_default_settings(exchange_connector)
+        self.set_connector_settings(exchange_connector)
+
+    @classmethod
+    def set_connector_settings(cls, exchange_connector) -> None:
+        """
+        override this method in the exchange tentacle to change default settings
+
+        for example:
+            self.MARKET_STATUS_PARSER.FIX_PRECISION = True
+            self.MARK_PRICE_IN_POSITION = True
+        """
+        pass
+
+    # set test status for each exchange
+
+    IS_FULLY_TESTED_AND_SUPPORTED = False  # not recommended
+    is_untested: exchange_test_status.ExchangeTestStatus = (
+        exchange_test_status.ExchangeTestStatus()
+    )
+    CANDLE_LOADING_LIMIT_TEST_STATUS: exchange_test_status.ExchangeTestStatus = is_untested
+    MAX_RECENT_TRADES_PAGINATION_LIMIT_TEST_STATUS: exchange_test_status.ExchangeTestStatus = is_untested
+    MAX_ORDER_PAGINATION_LIMIT_TEST_STATUS: exchange_test_status.ExchangeTestStatus = is_untested
+
+    MARKET_STATUS_TEST_STATUS: exchange_test_status.ExchangeTestStatus = is_untested
+    ORDERS_PARSER_TEST_STATUS: exchange_test_status.ExchangeTestStatus = is_untested
```

Comment:
Why do we need this kind of granularity in statuses, why not just "spot_tested" and "future_tested"? We may be more precise with "spot_auth_tested" but I don't think we should say if the market status has been tested or not